src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Components/WindowProcess.cs

```diff
@@ -177,6 +224,28 @@ internal static string GetProcessNameFromProcessID(uint pid)
             }
         }
 
+        /// <summary>
+        /// Gets the process file name for the process ID
+        /// </summary>
+        /// <param name="pid">The id of the process</param>
+        /// <returns>A string representing the file name or an empty string if the function fails</returns>
+        internal static string GetProcessFileNameFromID(uint pid)
```

Comment:
I would rename it to something like `GetProcessFilePathFromID` to make the difference to `GetProcessNameFromID` more clear.